lib/packwerk/cli/command_registry.rb

```diff
@@ -0,0 +1,64 @@
+# typed: strict
+# frozen_string_literal: true
+
+module Packwerk
+  class Cli
+    class CommandRegistry
```

Comment:
The instance of this class is an entry, not a registry, so it doesn't really make sense.  I would say that a better api would be `Commands.all` / `Commands.register`, and then name this class `Commands::Info`, `Commands::Lookup`, or `Commands::Entry` (etc.) so the registry can live on `Commands`.

Reply:
Yeah, the naming didn't please me. I'll have a look.

Reply:
I renamed `CommandRegistry` to `LazyLoadedCommand` (to make the distinction with `BaseCommand`). [d49a7ea](https://github.com/Shopify/packwerk/pull/351/commits/d49a7eaa4726d31a2013be5002b01d1ebf561444)

Reply:
> and then name this class `Commands::Info`, `Commands::Lookup`

Just understood what you meant here. Could work too. 🤔

Reply:
670fc67